xbmc/filesystem/NFSFile.cpp

```diff
@@ -432,11 +444,13 @@ void CNfsConnection::keepAlive(const std::string& _exportPath, struct nfsfh* _pF
   if (!pContext)// this should normally never happen - paranoia
     pContext = m_pNfsContext;
 
-  CLog::Log(LOGINFO, "NFS: sending keep alive after {} s.",
-            std::chrono::duration_cast<std::chrono::seconds>(KEEP_ALIVE_TIMEOUT).count());
+  CLog::LogF(LOGDEBUG, "sending keep alive after {}s.",
```

Comment:
I imagine you changed this to debug because it's a bit spammy with the faster keep alive.

Reply:
True

Reply:
I want to prevent someone from opening a new issue "log spamming when video is paused"  and you assigning it to me. :)